src/admin/webpack.config.babel.js

```diff
@@ -0,0 +1,90 @@
+import config from 'config';
+import { DefinePlugin } from 'webpack';
+import ExtractTextPlugin from 'extract-text-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import { resolve } from 'path';
+
+export default {
+    entry: {
+        index: [
+            resolve(__dirname, './js/main.js'),
+        ],
+    },
+    module: {
+        loaders: [{
+            test: /\.jsx?$/,
+            exclude: /node_modules/,
+            loader: 'babel-loader',
+            // Options to configure babel with
+            query: {
+                cacheDirectory: true,
+                plugins: [
+                    'transform-react-jsx',
+                    ['transform-runtime', {
+                        polyfill: false,
+                        regenerator: true,
+                    }],
+                    'add-module-exports',
```

Comment:
I'm in favor of keep this config, but what about move it in the `.babelrc` file?